examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,225 @@
+# Deploying Kubeflow with AWS EFS as Persistent Storage
+
+This guide describes how to deploy Kubeflow on AWS EKS using EFS as the Persistent Storage. <complete intro>
+
+## 1.0 Prerequisites
+
+1. Install the CLI tools
+2. Clone this repo and checkout the right branch
+3. Create an EKS cluster and
```

Comment:
You should look at the existing README and provide precise steps to follow and continue here after that.